docs/_blog/2025-12-17/biohackathon-germany.md

```diff
@@ -0,0 +1,35 @@
+---
+title:  "RO-Crate at the 4th BioHackathon Germany"
+author: ["Phil Reed"]
+date: 2025-12-17
+---
+
+The [4th BioHackathon Germany](https://www.denbi.de/de-nbi-events/1840-4th-biohackathon-germany) took place from 1-5 December 2025 in Walsrode, Germany, organised by de.NBI and ELIXIR Germany. 
+Life scientists, data managers, software developers and project leaders attended the hybrid event to work together on open source code, standards and infrastructure to advance research data pratices and tools.
+
+{% include image.html src="umbrellas.jpg" caption="BioHackathon Germany was hosted at the ANDERS Hotel Walsrode." alt="Colourful umbrellas mounted the ceiling as art." %}
+
+## Improving RO-Crate support in federated storage systems
+
+One of the hackathon projects was titled [Enhancing FAIR (Meta-)Data Practices in Life Science by Improving RO-Crates Support in Federated Storage Systems](https://das-abroxas.github.io/2025_Biohackathon_Documentation/project_description.html). 
+It was led by Sebastian Beyvers and Jannis Schlegel (University of Giessen).
+
+During their participation in the Biohackathon Germany 2025, their working group engaged in extensive discussions focused on the practical and technical challenges of integrating RO-Crates in federated storage systems for life science data. 
+The following sections present the central questions and ideas that emerged from these collaborative sessions. 
```

Comment:
I see these are summarised below, would just change the word "sections" as I assumed there would be more detail under section headings